cmd/expiration-mailer/main.go

```diff
@@ -44,16 +44,76 @@ type regStore interface {
 	GetRegistration(ctx context.Context, req *sapb.RegistrationID, _ ...grpc.CallOption) (*corepb.Registration, error)
 }
 
+// limiter tracks how many mails we've sent to a given address in a given day.
+// Note that this does not track mails across restarts of the process.
+type limiter struct {
+	sync.RWMutex
+	// currentDay is a day in UTC, truncated to 24 hours. When the current
+	// time is more than 24 hours past this date, all counts reset and this
+	// date is updated.
+	currentDay time.Time
+
+	// counts is a map from address to number of mails we have attempted to
+	// send during `currentDay`.
+	counts map[string]int
+
+	// limit is the number of sends after which we'll return an error from
+	// check()
+	limit int
+
+	clk clock.Clock
+}
+
+// expects limiter is locked
```

Comment:
Nit; comment format.

```suggestion
// expects limiter is locked
```